setup.py

```diff
@@ -62,7 +62,7 @@
 
 # Only import these if the above checks are okay
 # to avoid masking the real problem with import error.
-from setuptools import setup  # noqa
 from extension_helpers import get_extensions  # noqa
+from setuptools import setup  # noqa
```

Comment:
I don't think this order. setuptools is provided by Python org, but extension_helpers is our own thing. So setuptools should be listed first. Yes, I am nitpicking with a bot.

Reply:
I added `extension_helpers` to `known_first_party` so know `setuptools` does come first!